File: scope104/src/main/java/com/petro/scope104/data/db/dao/CountryDao.java
package com.petro.scope104.data.db.dao;

import androidx.room.Dao;
import androidx.room.Insert;
import androidx.room.OnConflictStrategy;

import com.petro.scope104.data.db.entity.CountryEntity;

import java.util.List;

@Dao
public interface CountryDao {
    @Insert(onConflict = OnConflictStrategy.REPLACE)
    void insert(List<CountryEntity> countries);
}

File: scope104/src/main/kotlin/com/petro/scope104/data/db/dao/CountryDao.kt
package com.petro.scope104.data.db.dao

import androidx.room.Dao
import androidx.room.Insert
import androidx.room.OnConflictStrategy
import com.petro.scope104.data.db.entity.CountryEntity

@Dao
interface CountryDao {
    @Insert(onConflict = OnConflictStrategy.REPLACE)
    fun insert(countries: List<CountryEntity?>?)
}